BaseLib/FileTools.cpp

```diff
@@ -62,7 +62,7 @@ void truncateFile( std::string const& filename)
  * found.
  */
 static
-std::size_t findLastPathSeparator(std::string const& path)
+std::string::size_type findLastPathSeparator(std::string const& path)
```

Comment:
This function is declared/defined only in the cpp file. I moved it to an anonymous namespace.